Packs/AcalvioShadowplex/Integrations/acalvioapp/acalvioapp.py

```diff
@@ -0,0 +1,533 @@
+                outputs=out_result,
+                readable_output=tableToMarkdown
+                ('Acalvio ShadowPlex - Unmute Deception Host', out_result),
+                raw_response=res.json()
+            )
+        else:
+            error = get_acal_error(res)
+
+    except DemistoException as de:
+        raise Exception(de)
+
+    return (results, error)
+# end of function - do_unmute_deception_host_command
+
+
+def do_mute_deception_ep_command(client, args):
+
+    results = None
+    error = None
+
+    try:
+        ep = args.get('endpoint')
+
+        res = client.call_acal_api(action=Action.MUTE_DECEPTION_EP,
+                                   ep=ep)
+
+        if 200 == res.status_code \
+                and 'rescode' in res.json():
+
+            out_result = {
+                'is_mute': True if 0 == res.json()['rescode'] else False,
+                'endpoint': str(ep)
+            }
+
+            results = CommandResults(
+                outputs_prefix='Acalvio.MuteDeceptionEndpoint',
+                outputs_key_field=['endpoint', 'is_mute'],
+                outputs=out_result,
+                readable_output=tableToMarkdown
+                ('Acalvio ShadowPlex - Mute Deception on Endpoint',
+                 out_result),
+                raw_response=res.json()
+            )
+        else:
+            error = get_acal_error(res)
+
+    except DemistoException as de:
+        raise Exception(de)
+
+    return (results, error)
+# end of function - do_mute_deception_ep_command
+
+
+def do_unmute_deception_ep_command(client, args):
+
+    results = None
+    error = None
+
+    try:
+        ep = args.get('endpoint')
+
+        res = client.call_acal_api(action=Action.UNMUTE_DECEPTION_EP,
+                                   ep=ep)
+
+        if 200 == res.status_code \
+                and 'rescode' in res.json():
+
+            out_result = {
+                'is_unmute': True if 0 == res.json()['rescode'] else False,
+                'endpoint': str(ep)
+            }
+
+            results = CommandResults(
+                outputs_prefix='Acalvio.UnmuteDeceptionEndpoint',
+                outputs_key_field=['endpoint', 'is_unmute'],
+                outputs=out_result,
+                readable_output=tableToMarkdown
+                ('Acalvio ShadowPlex - Unmute Deception on Endpoint',
+                 out_result),
+                raw_response=res.json()
+            )
+        else:
+            error = get_acal_error(res)
+
+    except DemistoException as de:
+        raise Exception(de)
+
+    return (results, error)
+# end of function - do_unmute_deception_ep_command
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+
+    # get Acalvio API Server url
+    base_url = demisto.params()['url']
+
+    # get Acalvio API Key
+    apikey = demisto.params()['apikey']
+
+    # check if SSL is to be verified
+    verify_certificate = demisto.params().get('insecure', False)
```

Comment:
wish to keep False by default

Reply:
It will remain as false by default. the verify_certificate has an opposite value of the insecure (insecure = true -> verify = false)
Please change as suggested.

Reply:
done